tasks/zaproxy.js

```diff
@@ -387,12 +388,25 @@ module.exports = function (grunt) {
 
   grunt.registerTask('zap_results', 'It fails if found an alert that should not be ignored', function () {
     var done = this.async();
+
+    var options = this.options({
+      risks : ['High', 'Medium', 'Low', 'Informational']
+    });
+
     grunt.log.write('Verifying alert errors...');
-    if (grunt.config.get('zap_alert.failed')) {
-      grunt.log.write('Error found!');
+
+    var alerts = grunt.config.get('zap_alert.alerts') || [];
+
+    var filteredAlerts = alerts.filter(function(alert){
+      return options.risks.indexOf(alert.risk) > -1;
```

Comment:
Didn't use _.contains from lodash, because it looks at the all properties of an object. So I preferred to use filter and only look at the risk property.